mantidproject/mslice
# Updated readme.md for using Pycharm

Updated README.md to allow setting up PyCharm correctly.

**To test:**

Setup PyCharm using the instructions from README.md and check that you can run MSlice.

Fixes #[590](https://github.com/mantidproject/mslice/issues/590).
